bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/ApiController.java

```diff
@@ -76,6 +76,7 @@ public class ApiController {
 
     private static final String HEADER_REMAINING_CALLS = "RemainingCalls";
     private static final String HEADER_TOTAL_CALLS = "TotalCalls";
+    private static final int REQUEST_TIMEOUT_MS = 10_000;
```

Comment:
Since the granularity of milliseconds is not used for this hardcoded constant, it could be expressed in seconds which would make it even more readable:
```suggestion
    private static final int REQUEST_TIMEOUT_SECONDS = 30;
```

Reply:
Done